services/initChatAnonymous/initChatAnonymousServices.ts

```diff
@@ -52,7 +52,7 @@ type InitChatAnonymousPayload = {
   anon_user_info_color_name: string;
   anon_user_info_emoji_code: string;
   anon_user_info_emoji_name: string;
-  member: string[];
+  member: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from an array of strings to a single string for the `member` property might cause issues if multiple members are expected in some scenarios. Please ensure that this change aligns with all use cases of the `member` property throughout your application.
```diff
-  member: string[];
+  member: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->